src/encord_active/analysis/metrics/image/contrast.py

```diff
@@ -0,0 +1,57 @@
+from typing import Optional, cast
+
+import torch
+
+from encord_active.analysis.metric import (
+    ImageObjectOnlyOutputBatch,
+    MetricDependencies,
+    ObjectOnlyBatchInput,
+    OneImageMetric,
+)
+from encord_active.analysis.types import (
+    BoundingBoxTensor,
+    ImageBatchTensor,
+    ImageTensor,
+    MaskTensor,
+    MetricBatchDependencies,
+    MetricResult,
+)
+from encord_active.db.metrics import MetricType
+
+
+class ContrastMetric(OneImageMetric):
+    def __init__(self) -> None:
+        super().__init__(
+            ident="metric_contrast",
+            long_name="Contrast",
+            desc="",
+            metric_type=MetricType.NORMAL,
+        )
+
+    def calculate(
+        self, deps: MetricDependencies, image: ImageTensor, mask: Optional[MaskTensor], bb: Optional[BoundingBoxTensor]
+    ) -> MetricResult:
+        # Max-std : [0, 255] = 255/2 = 127.5
+        grayscale_image = cast(torch.Tensor, deps["ephemeral_grayscale_image"])
+        return torch.std(grayscale_image.type(torch.float32)).cpu() / 127.5
```

Comment:
We may need to make this change:

```suggestion
        return torch.std(grayscale_image.type(torch.float32), correction=0).cpu() / 127.5
```

according to [here](https://pytorch.org/docs/stable/generated/torch.std.html)